sycl/include/CL/sycl/detail/queue_impl.hpp

```diff
@@ -127,39 +127,31 @@ class queue_impl {
   }
 
   RT::PiQueue createQueue() {
-    cl_command_queue_properties CreationFlags = 0;
+    pi_uint64 CreationFlags = 0;
```

Comment:
Not sure if there is a more PI friendly way of doing the bit operations with flags on an enumerator, currently used a pi_uint64 as there is no equivalent to cl_bitfield (which is just a typedef to cl_ulong) in the PI interface and I'm unsure if we intend to have a PI equivalent for it.

Reply:
And since all this PI story has probably some deep motivation for Intel we are completely ignorant about, it is difficult to contribute on this side without seeing the big picture...

Reply:
Could you actually change the pi_queue_properties enum to pi_uint64 in pi.h similar to how pi_mem_flags is done. Sorry for misleading initial comment about enum (they are not good where bit manipulations are needed).

Reply:
Sure, not a problem. Happy to oblige!

Reply:
@smaslov-intel  Could we have an alias `pi_bitfield` for pi_uint64 to make the code clearer?

Reply:
NP, let's do that if that's clearer.

Reply:
probably for a separate PR or would you like me to make the change in this PR?

Reply:
Isn't this a simple typdef, and few uses? It can fit this PR, but I am also fine doing so in a separate one.

Reply:
I imagine it is, not looked into it just yet. I just thought it may be relevant to have it in a differently titled pull request for histories sake. Are we happy to use the existing pi_uint64 as the initial type for the pi_bitfield type?